src/website/app/ComponentDocExample.js

```diff
@@ -90,7 +95,9 @@ export default class ComponentDocExample extends Component {
     return (
       <Root className={className} id={id}>
         <Title level={3} id={id}>
-          {title}
+          <TitleLink to={`./${id}`}>
```

Comment:
I really like how the title links to the standalone example, while the # links to the target without leaving the page. 👍

Reply:
Glad you like that pattern! I was debating a few different tacts.